resources/splunk_install.rb

```diff
@@ -84,6 +85,13 @@ def post_install
     user_resource.manage_home true
     user_resource.run_action :create
 
+    if group != user
```

Comment:
Ah, I didn't realize system users automatically create a matching group.  Good to know, thanks.